### PR TITLE
fix syntax in commit 487e00d6

### PR DESCRIPTION
**Pull request by xutian**
Signed-off-by: Xu Tian <xutian@redhat.com>

### CODE REVIEW

**Comment by xutian** [comment]
@yanan-fu @luckyh   hotfix for merged 487e00d6. would you like to help review.

**Comment by yanan-fu** [comment]
@xutian   i have send another patch to fix this isse yesterday:
https://github.com/avocado-framework/avocado-vt/pull/2419

File: virttest/qemu_devices/qcontainer.py [hunk 1]
```diff
@@ -2180,6 +2180,7 @@ def serials_define_by_variables(self, serial_id, serial_type, chardev_id,
         """
 
         devices = []
+        bus_params = {}
         if bus_extra_params:
             bus_params = (dict([_.split('=')
```
Comment (yanan-fu) [inline virttest/qemu_devices/qcontainer.py]: `bus_params` only available when `serial_type.startswith('virt')`.

**Comment by xutian** [comment]
yanan-fu  has a hotfix, so let's close this one. thanks @luckyh  and @yanan-fu

**Pull request closed without merging**